Azure-Samples/fastapi-on-azure-functions
# Upgrade to Python programming model v2

## Purpose

This PR updates this sample to use the new Python programming model.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Follow README instructions for running locally
* Run azd up and verify routes respond
